kotlin-muix-tree-view/src/jsMain/generated/muix/tree/view/SimpleTreeView.types.kt

```diff
@@ -41,6 +41,13 @@ external interface SimpleTreeViewProps :
      * The ref object that allows Tree View manipulation. Can be instantiated with `useTreeViewApiRef()`.
      */
     var apiRef: react.Ref<*>?
+
+    /**
+     * Unstable features, breaking changes might be introduced.
+     * For each feature, if the flag is not explicitly set to `true`,
+     * the feature will be fully disabled and any property / method call will not have any effect.
+     */
+    var experimentalFeatures: Any? /* TreeViewExperimentalFeatures<SimpleTreeViewPlugins> */
```

Comment:
`TreeViewExperimentalFeatures` looks like fine type

Reply:
Unfortunately no, it's complicated to calculate and it changes from release to release, so I decided to use `Any?`

Reply:
> it changes from release to release

It's expected :)